### PR TITLE
Fix nearby fakegms API to better match the real gms package

### LINKED ISSUE #56: Adjust fakegms.nearby API to match the real one
I found some differences between the real GMS types and the fakeGMS types, namely:

- Most classes expose fields as public and mutable instead of making the fields `private final` and exposing the values using a getter
  - Changing to getters will presumably look the same to Kotlin callers
- `ExposureConfiguration` is missing all weight parameters (is this intentional?)
- ExposureNotificationStatusCodes is an enum instead of an uninstantiable class with `public static final int` status codes
  - This is probably not a huge deal, but it's limiting compatibility
- Some classes haven't actually been implemented

### PR DESCRIPTION
**Pull request by BjoernPetersen**
Fixes #56 (see issue description)

### CODE REVIEW

File: sdk/src/main/java/org/coralibre/android/sdk/fakegms/nearby/exposurenotification/ExposureConfiguration.java [hunk 1]
```diff
@@ -227,12 +272,12 @@ public int getTransmissionRiskScore(final int transmissionRiskValue) {
...
-     *                         {@code transmission power - } {@link CapturedData#getRssi()}
+     * @param attenuationValue           the bluetooth attenuation value from an exposure, in dB:
+     *                                   {@code transmission power - } {@link CapturedData#getRssi()}
      * @param daysSinceLastExposureValue the number of days since the last exposure
-     * @param durationValue the duration of an exposure, in minutes
-     * @param transmissionRiskValue the user defined risk value associated to an exposure,
-     *                              must be >= 0 and <= 7
+     * @param durationValue              the duration of an exposure, in minutes
+     * @param transmissionRiskValue      the user defined risk value associated to an exposure,
+     *                                   must be >= 0 and <= 7
```
Comment (Stypox) [inline sdk/src/main/java/org/coralibre/android/sdk/fakegms/nearby/exposurenotification/ExposureConfiguration.java:280]: Why adding those spaces? Did the IDE do it automatically? In my humble opinion it looked better as it was before, though feel free to ignore me ;-)

File: sdk/src/main/java/org/coralibre/android/sdk/fakegms/nearby/exposurenotification/ExposureSummary.java [hunk 2]
```diff
@@ -12,104 +17,200 @@
...
+     * low threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[0])<br>
      * attenuationDurations[1]  |  sum of durations of exposures with an attenuation greater than
-     *                             equal to the low threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[0])
-     *                             but less than the high threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[1])<br>
+     * equal to the low threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[0])
+     * but less than the high threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[1])<br>
      * attenuationDurations[2]  |  sum of durations of exposures with an attenuation greater than
-     *                             or equal to the high threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[1])<br>
+     * or equal to the high threshold (i.e. {@link ExposureConfiguration#durationAtAttenuationThresholds}[1])<br>
+     *
```
Comment (Stypox) [inline sdk/src/main/java/org/coralibre/android/sdk/fakegms/nearby/exposurenotification/ExposureSummary.java:118]: Add back the spaces, these were here on purpose

**Review by Stypox — COMMENTED** [review]
Looks good to me, apart from the javadoc indentation changes ;-)

File: sdk/src/main/java/org/coralibre/android/sdk/fakegms/nearby/exposurenotification/ExposureSummary.java (hunk 2 above)
Comment (BjoernPetersen) [reply sdk/src/main/java/org/coralibre/android/sdk/fakegms/nearby/exposurenotification/ExposureSummary.java]: Spaces – or whitespace in general – don't translate well in JavaDoc. If anybody has to read the JavaDoc for that function without looking at the comment in the source code it looks like this: 
Screen Shot 2020-09-29 at 11 03 11

I think we can both agree that that's also not the intended layout. I tried to make it readable both in "compiled" JavaDoc and the source code by converting it to a table, are you okay with the new format?

Comment (BjoernPetersen) [reply sdk/src/main/java/org/coralibre/android/sdk/fakegms/nearby/exposurenotification/ExposureSummary.java]: BTW the new version would look like this:

Screen Shot 2020-09-29 at 11 13 15

Comment (Stypox) [reply sdk/src/main/java/org/coralibre/android/sdk/fakegms/nearby/exposurenotification/ExposureSummary.java]: Ok, great, thank you

**Comment by Stypox** [comment]
LGTM :-D

**Review by Stypox — APPROVED** [review]
LGTM :-D

**Pull request merged**